presto-hive/src/main/java/com/facebook/presto/hive/metastore/MetastoreUtil.java

```diff
@@ -111,7 +111,10 @@ private static Properties getHiveSchema(
         schema.setProperty(META_TABLE_LOCATION, sd.getLocation());
 
         if (sd.getBucketProperty().isPresent()) {
-            schema.setProperty(BUCKET_FIELD_NAME, sd.getBucketProperty().get().getBucketedBy().get(0));
+            List<String> bucketedBy = sd.getBucketProperty().get().getBucketedBy();
+            if (!bucketedBy.isEmpty()) {
+                schema.setProperty(BUCKET_FIELD_NAME, bucketedBy.get(0));
```

Comment:
nit: do we want to use `getOnlyElement` ? -- although it's an independent improvement to the code :)

Reply:
That's a weird thing about this code. Even if there more than a single bucket column - this property is set only to the first bucketed column (at least in the version of Hive libraries that presto-hive depends on). This is apparently a bug, and was fixed in the Hive's downstream: https://github.com/apache/hive/commit/bf8d305a6fd435b3b510dc3f78b5f3329acbe423. However I can't be sure that there will be no consequences if we changed that without changing the Hive library. So i decided that it is better not to touch it.